applications_prd/argocd.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: argocd
+  namespace: argocd
+  labels:
+    purpose: infra
+spec:
+  destination:
+    namespace: argocd
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: ionos_prd/argocd
+    repoURL: https://github.com/DOME-Marketplace/dome-gitops
+    targetRevision: env_production
```

Comment:
replace env_production with HEAD (in all application files)

Reply:
I am going to do that directly before the merge as currently it would damage the environment